tasks/date-manipulation/src/main.rs

```diff
@@ -0,0 +1,19 @@
+use chrono::prelude::*;
+use chrono::Duration;
+
+fn main() {
+    // cannot parse TZ unfortunately
```

Comment:
Let's expand on this a bit:

```
// Chrono allows parsing time zone abbreviations like "EST", but
// their meaning is ignored due to a lack of standardization.
//
// This solution compromises by augmenting the parsed datetime
// with the timezone using the IANA abbreviation.
```

Reply:
Thanks, I have edded these comments and rebased